crates/control/src/ball_filter.rs

```diff
@@ -258,21 +263,21 @@ impl BallFilter {
 
     fn update_hypothesis_with_measurement(
         hypothesis: &mut Hypothesis,
-        detected_position: Point2<f32>,
+        detected_position: Point2<Ground>,
         detection_time: SystemTime,
         configuration: &BallFilterParameters,
     ) {
         hypothesis.moving_state.update(
             Matrix2x4::identity(),
-            detected_position.coords,
+            detected_position.inner.coords,
```

Comment:
I thought we agreed not to frame the multivariate distributions. Because of that, we need a raw nalgebra vector here, either through `.coords().inner` or through `.inner.coords`.

Reply:
sorry, mixed that up. resolving...